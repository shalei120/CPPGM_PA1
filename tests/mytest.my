new-line 0 
non-whitespace-character 1 \
identifier 3 u34
new-line 0 
pp-number 5 2342_
utf8 segmentation error!
ERROR: utf8 segmentation error!

