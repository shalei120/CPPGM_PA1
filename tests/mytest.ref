new-line 0 
non-whitespace-character 1 \
identifier 3 u34
new-line 0 
pp-number 10 2342_㐒er
new-line 0 
character-literal 4 'π'
new-line 0 
new-line 0 
preprocessing-op-or-punc 1 ?
preprocessing-op-or-punc 1 ?
preprocessing-op-or-punc 1 #
new-line 0 
string-literal 25 "dghsdf/*dsfgadfg*/gsdfg"
new-line 0 
new-line 0 
preprocessing-op-or-punc 1 #
identifier 6 define
whitespace-sequence 0 
identifier 1 R
whitespace-sequence 0 
string-literal 3 "x"
new-line 0 
identifier 5 const
whitespace-sequence 0 
identifier 4 char
preprocessing-op-or-punc 1 *
whitespace-sequence 0 
identifier 1 s
preprocessing-op-or-punc 1 =
