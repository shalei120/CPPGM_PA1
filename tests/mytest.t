
\u34
2342_\u3412er
'Ï€'

??#
"dghsdf/*dsfgadfg*/gsdfg"

#define R "x"
const char* s=R"y";
