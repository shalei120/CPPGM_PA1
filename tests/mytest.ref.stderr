ERROR: invalid characters in raw string delimiter
